src/VisualStudio/CSharp/Impl/Options/AutomationObject.cs

```diff
@@ -194,6 +194,18 @@ public int NewLines_Braces_Method
             set { SetBooleanOption(CSharpFormattingOptions.NewLinesForBracesInMethods, value); }
         }
 
+        public int NewLines_Braces_Property
+        {
+            get { return GetBooleanOption(CSharpFormattingOptions.NewLinesForBracesInProperties); }
+            set { SetBooleanOption(CSharpFormattingOptions.NewLinesForBracesInProperties, value); }
+        }
+
+        public int NewLines_Braces_Accessor
```

Comment:
Yeah, I waffled a bit.  As the one who added most of the options VS has back in VS2005, I know about them not being used much, but this seemed like such an obvious next step I just did it.